README.md

```diff
@@ -1,723 +0,0 @@
-Jupyter notebook extensions
```

Comment:
You seem to have mistakenly removed the main `README.md` - could we get this reinstated?